elastic/kibana
# [embeddable rebuild] log stream react embeddable

PR migrates log stream embeddable from the legacy class based system.

### test instructions
1. Run kibana on a system with o11y data and log streams
2. Create a new dashboard, click "Add panel" => "Log stream"
3. Verify panel behavior has not changed with legacy embeddable
4. Click panel context menu and select "Settings"
5. Set custom title, description and time range. Verify behavior has not changed with legacy embeddable 
6. Import dashboard with log stream panel. Verify behavior has not changed with legacy embeddable
